index.html

```diff
@@ -36,7 +147,45 @@ <h1>I’m Shahnawaz Ali
                 <li><a href="#"><img src="./images/images6.svg" alt="medium"></a></li>
 
             </ul>
+            <button class="btn2" type="submit">See Project</button>
```

Comment:
- [ ] The text on the button here is `Get my resume`. Kindly change the text to match the template